docs/tutorials/bigquery.ipynb

```diff
@@ -0,0 +1,984 @@
+        "  </td>\n",
+        "      <td>\n",
+        "    <a href=\"https://raw.githubusercontent.com/tensorflow/io/master/docs/tutorials/bigquery.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "xHxb-dlhMIzW"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "This tutorial shows how to use BigQuery TensorFlow reader for training neural network using the Keras sequential API."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WodUv8O1VKmr",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Dataset\n",
+        "\n",
+        "This tutorial uses the [United States Census Income\n",
+        "Dataset](https://archive.ics.uci.edu/ml/datasets/census+income) provided by the\n",
+        "[UC Irvine Machine Learning\n",
+        "Repository](https://archive.ics.uci.edu/ml/index.php). This dataset contains\n",
+        "information about people from a 1994 Census database, including age, education,\n",
+        "marital status, occupation, and whether they make more than $50,000 a year."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4YsfgDMZW5g6",
+        "colab_type": "text"
+      },
+      "source": [
+        "Set up your GCP project\n",
+        "\n",
+        "**The following steps are required, regardless of your notebook environment.**\n",
+        "\n",
+        "1. [Select or create a GCP project.](https://console.cloud.google.com/cloud-resource-manager)\n",
+        "\n",
+        "2. [Make sure that billing is enabled for your project.](https://cloud.google.com/billing/docs/how-to/modify-project)\n",
+        "\n",
+        "3. [Enable the BigQuery Storage API](https://cloud.google.com/bigquery/docs/reference/storage/#enabling_the_api)\n",
+        "\n",
+        "4. Enter your project ID in the cell below. Then run the  cell to make sure the\n",
+        "Cloud SDK uses the right project for all the commands in this notebook.\n",
+        "\n",
+        "**Note**: Jupyter runs lines prefixed with `!` as shell commands, and it interpolates Python variables prefixed with `$` into these commands."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "Install required Packages, and restart runtime"
```

Comment:
I'm a little unclear what's required to install before restarting the runtime.

In general, I think it would be useful to separate the Google Cloud imports from the TensorFlow/generic Python imports since I always find GCP a bit confusing :P